source/_posts/2020-11-12-android-300-release.markdown

```diff
@@ -0,0 +1,128 @@
+- Last Update Trigger - A sensor whose state will represent the reason for the last update that was sent to your Home Assistant instance. This sensor will update anytime an update is sent to your Home Assistant instance.
+
+## Sensor Settings
+
+We didn't stop at just adding new sensors, we also made enhancements to the overall sensor experience. Starting from version 2.5.0 certain sensors have custom settings that can help with what updates actually get sent to your Home Assistant instance.
+
+
+<p class='img'>
+<img src='/images/blog/2020-11-05-android-300-release/location_sensor_settings.png' alt='Screenshot of location settings'></a>
+Screenshot of location settings.
+</p>
+
+- Next Alarm - This sensor has a setting for an allow list. This means that if you have an app that reports really odd timestamps as an actual alarm, you can now ignore them by telling the app which packages to send reports from. By default the list is blank. Tasker users are recommended to make use of this setting.
+- Last Notification - This sensor also has an allow list to let the user create a list of apps from which they want notification data from. By default all notifications are sent to your Home Assistant server. We highly recommend that you setup an allow list as soon as you can think of one to prevent a high amount of updates. You will be surprised by the amount of data that will show up in a short time period.
+- Last Reboot - A bug was discovered where sometimes the timing in the calculation of the device's last reboot could be off causing an unnecessary update. There is now a deadband setting to allow you to adjust the timing to ignore updates. By default this is set to 1 minute, you most likely won't need to change this.
+- Location Sensors - All 3 location sensors now have settings to allow you to adjust the Minimum Accuracy required to send a update to your Home Assistant server. There is also a setting to adjust the minimum amount of time between updates to get sent. This should help a lot of users out who don't get the location results they expect. We recommend changing this setting after you evaluate all of the location fixes in 3.0.0 as location tracking may already be improved without needing to adjust these.
+- WiFi BSSID - This sensor has a setting that will allow the user to rename the currently connected BSSID. Not everyone can remember a mac address let alone dozens of them. This setting is designed to help out users who use this sensor to make better sense of things without the need for secrets or templates. If you live in a household with multiple access points you may find it useful to rename a BSSID to help with things like room presence. By default this sensor reports the connected mac address.
```

Comment:
```suggestion
- WiFi BSSID - This sensor has a setting that will allow the user to provide an alias for the currently connected BSSID. Not everyone can remember a MAC address let alone dozens of them. This setting is designed to help out those who use this sensor to make better sense of things without the need for secrets or templates. If you live in a household with multiple access points you may find it useful to set up an alias to help with things like room presence. By default this sensor reports the connected MAC address.
```
Renaming a BSSID could be seen to be actually changing it. Nice feature too.

"users who use" just sounds odd

Reply:
alias is much better, thank you :)